.github/container/Dockerfile.mjx

```diff
@@ -47,4 +50,8 @@ EOF
 
 FROM mealkit as final
 
-RUN pip-finalize.sh
+# compile MuJoCo-MPC with Clang separately 
+RUN CC=${CC_COMPILER} CXX=${CXX_COMPILER} pip-finalize.sh /opt/pip-tools.d/requirements-mpc.in
+
+# compiler the rest dependencies
+RUN pip-finalize.sh /opt/pip-tools.d/requirements-mjx.in /opt/pip-tools.d/requirements-l2r.in
```

Comment:
Why do we need to run `pip-finalize` twice here? Due to the use of `pip-sync` in the script, the second run will erase what the first run installed.